### PR TITLE
fix: style fixes

### PR DESCRIPTION
**Pull request by itssharmasandeep**
## Description
Style fixes

### CODE REVIEW

File: projects/components/src/list-view/list-view.component.scss [hunk 1]
```diff
@@ -34,7 +34,7 @@ $margin-left: 12px;
     .header-key-label,
     .header-value-label {
       @include overline;
-      margin-left: $margin-left;
+      padding: 0px $horizontal-offset;
```
Comment (arjunlalb) [inline projects/components/src/list-view/list-view.component.scss:37]: make these `padding: 0 $horizontal-offset`

0 doesn't need units.

File: projects/distributed-tracing/src/shared/components/log-events/log-events-table.component.scss [hunk 2]
```diff
@@ -1,7 +1,13 @@
 @import 'font';
 
 .log-events-table {
-  margin-top: 25px;
+  padding-top: 25px;
```
Comment (arjunlalb) [inline projects/distributed-tracing/src/shared/components/log-events/log-events-table.component.scss:4]: we use spacing in multiple of 4. what does the mock say here? perhaps 24 ?

Comment (itssharmasandeep) [reply projects/distributed-tracing/src/shared/components/log-events/log-events-table.component.scss]: Yeah,Fixed!!

File: projects/components/src/list-view/list-view.component.scss (hunk 1 above)
Comment (itssharmasandeep) [reply projects/components/src/list-view/list-view.component.scss]: yeah, Fixed!!

**Review by arjunlalb — APPROVED** [review]

**Pull request merged**